README.md

```diff
@@ -1,3 +1,37 @@
+<img src="https://cdn.miranj.in/mc3/img/apple-touch-icon-precomposed.png">
+
+## Miranj Craft Boilerplate
+
+This is [Miranj's](https://miranj.in) Craft CMS boilerplate. It includes:
+
+* Server configuration files for Apache and nginx that enable clean URLs and high performance.
+* A Craft CMS configuration that lets many configuration details be controlled by [environment variables](https://en.wikipedia.org/wiki/Environment_variable),
+and sets the rest to the standards used at Miranj.
+* A [gulp](https://gulpjs.com/) based build process that uses [Tailwind CSS](https://tailwindcss.com/docs/what-is-tailwind/)
+with easy custom configuration of styling.
+* Miranj's preferred Craft plugins, as well as custom configurations for the same:
+    - [Minify](https://github.com/nystudio107/craft-minify)
+    - [Imager](https://github.com/aelvan/Imager-Craft)
+    - [Expanded Singles](https://github.com/verbb/expanded-singles)
+    - [Redactor](https://github.com/craftcms/redactor)
+    - [Inventory](https://github.com/doublesecretagency/craft-inventory)
+    - [Environment Label](https://github.com/TopShelfCraft/Environment-Label)
+    - [Typogrify](https://github.com/nystudio107/craft-typogrify)
+    - [Relabel](https://github.com/Anubarak/craft-relabel)
+    - [Element Index Defaults](https://github.com/verbb/element-index-defaults)
+    - [CacheFlag](https://github.com/mmikkel/CacheFlag-Craft3)
+* Miranj's preferred JS utilities:
+    - [PJAX](https://github.com/MoOx/pjax)
+    - [lazySizes](https://github.com/aFarkas/lazysizes)
+    - [Web Font Loader](https://github.com/typekit/webfontloader)
+* And lastly our template hierarchy and organization.
```

Comment:
This should perhaps be linked to [Souvik’s presentation on modular architecture](https://miranj.in/blog/2019/modular-architecture-for-building-content-websites).

Reply:
Link added.